### PR TITLE
CLOUD-563

### PR DESCRIPTION
**Pull request by wstrange**
use git-init containers

### CODE REVIEW

File: helm/frconfig/values.yaml [hunk 1]
```diff
@@ -4,6 +4,13 @@
 
 replicaCount: 1
 
+
+config:
+  name: frconfig
```
Comment (dgoldssfo) [inline helm/frconfig/values.yaml]: So the way I read this is that users will still deploy the frconfig chart and it's basically a container for the Git secret and the details of the Git repository. 
Also, when redeploying AM/IDM/IG in the event of a config change, seems like there's no longer the need to reinstall the frconfig chart.
Could you confirm my understanding is correct? Thanks.

File: helm/amster/templates/amster.yaml [hunk 2]
```diff
@@ -20,7 +20,20 @@ spec:
         app: {{ template "fullname" . }}
         component:  {{ .Values.component }}
     spec:
-      terminationGracePeriodSeconds: 5    
+      terminationGracePeriodSeconds: 5   
+      initContainers:
+      - name: git-init
+        image: forgerock/git:6.0.0
```
Comment (wajihahmed) [inline helm/amster/templates/amster.yaml]: should this be 6.5.0?

File: bin/deploy.sh [hunk 3]
```diff
@@ -131,7 +131,7 @@ deploy_charts()
         chart="${comp}"
         case "${comp}" in
           *store)
-            chart="opendj"
+            chart="ds"
```
[the remark this answers is not in the document]
Comment (wstrange) [reply bin/deploy.sh]: You can start to use ds/

File: helm/frconfig/values.yaml (hunk 1 above)
Comment (wstrange) [reply helm/frconfig/values.yaml]: Correct. 
It also means that if they want different config repos (for say IG and IDM), they can deploy another instance of frconfig, with a different config name.

File: helm/amster/templates/amster.yaml (hunk 2 above)
Comment (wstrange) [reply helm/amster/templates/amster.yaml]: Eventually. Right now there have not been any changes

File: bin/deploy.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (dgoldssfo) [reply bin/deploy.sh]: I realize it's going to be a pain, but would you consider changing the names of the openam, openidm, and openig charts so we can have naming consistency? (I'd personally prefer opendj/openam/openidm/openig to ds/openam/openidm/openig)
Not for this PR obviously - I'll create a Jira if you'll consider doing it. Let me know.

**Pull request merged**